documentation/source/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/before-starting.rst

```diff
@@ -0,0 +1,15 @@
+Before starting this tutorial
+#############################
+
+To modify
+#. Make sure that you have the following files in your working directory:
+
+   * ``single_subject/data/t2_compression/t2_compression.nii.gz``: A 3D binary mask for the spinal cord of a T2 anatomical image with spinal cord compression.
```

Comment:
We could use the `label-compression.nii.gz` suffix to be in line with the [intranet](https://intranet.neuro.polymtl.ca/data/dataset-curation.html#derivatives-structure).

Reply:
This is the anatomical image, not the label! it is just that we have to name it differently since we already have` t2.nii.gz`

Reply:
Oops, sorry, my mistake.